models/issues/issue_project.go

```diff
@@ -16,13 +16,14 @@ import (
 func (issue *Issue) LoadProject(ctx context.Context) (err error) {
 	if issue.Project == nil {
 		var p project_model.Project
-		if _, err = db.GetEngine(ctx).Table("project").
+		has, err := db.GetEngine(ctx).Table("project").
```

Comment:
I think `has` as variable name is not ideal, what about something like `res` (for result/response)?

Reply:
1. `has` is almost idiomatic within the gitea code
2. `res` is not what's returned here: The function returns `bool, err` with exactly the meaning I described in 1.
3. The actual result is set within the function as a pointer is explicitly passed: The `&p` means that the variable `p` is changed directly.

Reply:
The only potential alternative to `has` that would make sense and is also used throughout the code is `exists` which is probably a better name for it.

Reply:
Oh okay, didn't know that it is used in Gitea 😆